bitsandbytes/autograd/_functions.py

```diff
@@ -519,7 +519,12 @@ def forward(ctx, A, B, out=None, bias=None, quant_state: Optional[F.QuantState]
 
         # 1. Dequantize
         # 2. MatmulnN
-        output = torch.nn.functional.linear(A, F.dequantize_4bit(B, quant_state).to(A.dtype).t(), bias)
+        if A.device.type == "npu":
```

Comment:
A quick question: why not we use `torch.nn.functional.linear` directly? Thanks in advance for your answer